### ISSUE TITLE
Install Java on the nodes

### ISSUE THREAD

**Issue by hemenkapadia** [hemenkapadia/vagrant-hadoop-cluster#4]
Depends on successful completion of issue #3 .

Install Java on the nodes - for now i am going with OpenJDK but we can also get code from vagrant-webdev project in case we need Oracle JDK installed.

**Comment by hemenkapadia** [comment]
Enables java model and re-provisioned the cluster.

``` Shellsession
hekapadi@HEKAPADI-W7-3 ~/Workspaces/Personal/vagrant-hadoop-cluster (hemen_initial)
$ vagrant ssh master.local
Welcome to Ubuntu 12.04.5 LTS (GNU/Linux 3.13.0-32-generic x86_64)

 * Documentation:  https://help.ubuntu.com/
New release '14.04.2 LTS' available.
Run 'do-release-upgrade' to upgrade to it.

Last login: Sun Oct 18 15:24:24 2015 from 10.0.2.2
vagrant@master:~$ echo $JAVA_HOME
/usr/lib/jvm/java-7-openjdk-amd64
vagrant@master:~$ java -version
java version "1.7.0_79"
OpenJDK Runtime Environment (IcedTea 2.5.6) (7u79-2.5.6-0ubuntu1.12.04.1)
OpenJDK 64-Bit Server VM (build 24.79-b02, mixed mode)
vagrant@master:~$ sudo su - hadoop
hadoop@master:~$ uname -a
Linux master 3.13.0-32-generic #57~precise1-Ubuntu SMP Tue Jul 15 03:51:20 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux
hadoop@master:~$ echo $JAVA_HOME
/usr/lib/jvm/java-7-openjdk-amd64
hadoop@master:~$ java -version
java version "1.7.0_79"
OpenJDK Runtime Environment (IcedTea 2.5.6) (7u79-2.5.6-0ubuntu1.12.04.1)
OpenJDK 64-Bit Server VM (build 24.79-b02, mixed mode)
hadoop@master:~$ ssh hadoop1.local
Welcome to Ubuntu 12.04.5 LTS (GNU/Linux 3.13.0-32-generic x86_64)

 * Documentation:  https://help.ubuntu.com/
New release '14.04.2 LTS' available.
Run 'do-release-upgrade' to upgrade to it.

Last login: Sun Oct 18 16:51:19 2015 from 192.168.48.10
hadoop@hadoop1:~$ echo $JAVA_HOME
/usr/lib/jvm/java-7-openjdk-amd64
hadoop@hadoop1:~$ java -version
java version "1.7.0_79"
OpenJDK Runtime Environment (IcedTea 2.5.6) (7u79-2.5.6-0ubuntu1.12.04.1)
OpenJDK 64-Bit Server VM (build 24.79-b02, mixed mode)
hadoop@hadoop1:~$ ssh hadoop2.local
Warning: Permanently added 'hadoop2.local,192.168.48.12' (ECDSA) to the list of known hosts.
Welcome to Ubuntu 12.04.5 LTS (GNU/Linux 3.13.0-32-generic x86_64)

 * Documentation:  https://help.ubuntu.com/
New release '14.04.2 LTS' available.
Run 'do-release-upgrade' to upgrade to it.

Last login: Sun Oct 18 14:02:15 2015 from 192.168.48.10
hadoop@hadoop2:~$ echo $JAVA_HOME
/usr/lib/jvm/java-7-openjdk-amd64
hadoop@hadoop2:~$ java -version
java version "1.7.0_79"
OpenJDK Runtime Environment (IcedTea 2.5.6) (7u79-2.5.6-0ubuntu1.12.04.1)
OpenJDK 64-Bit Server VM (build 24.79-b02, mixed mode)
hadoop@hadoop2:~$ ssh hadoop3.local
Warning: Permanently added 'hadoop3.local,192.168.48.13' (ECDSA) to the list of known hosts.
Welcome to Ubuntu 12.04.5 LTS (GNU/Linux 3.13.0-32-generic x86_64)

 * Documentation:  https://help.ubuntu.com/
New release '14.04.2 LTS' available.
Run 'do-release-upgrade' to upgrade to it.

Last login: Sun Oct 18 14:03:23 2015 from 192.168.48.10
hadoop@hadoop3:~$ echo $JAVA_HOME
/usr/lib/jvm/java-7-openjdk-amd64
hadoop@hadoop3:~$ java -version
java version "1.7.0_79"
OpenJDK Runtime Environment (IcedTea 2.5.6) (7u79-2.5.6-0ubuntu1.12.04.1)
OpenJDK 64-Bit Server VM (build 24.79-b02, mixed mode)
hadoop@hadoop3:~$
```

**Comment by hemenkapadia** [comment]
Closed as part of commit 519cdb02c5ca6224cbad0a615e066ec296e5a509 in hemen_inital branch
